### DISCUSSION TITLE
Get a single line closed polygon back from a fixed polygon

### DISCUSSION THREAD

**Discussion by Workings** [NetTopologySuite/NetTopologySuite#773] [Q&A]
Have a polygon which needed fixing (using either 'NetTopologySuite.Geometries.Utilities.GeometryFixer.Fix' or NetTopologySuite.Geometries.Geometry.Buffer(0)), but when trying to get points back from it its no longer a single valid closed polygon, but two separate internal and external polygons, with the combined Coordinates of the geometry not lining up anymore. Any method in NetTopologySuite to return a single valid closed polygon again?

**Comment by Workings** [comment]
Currently trying to manually join the external and internal rings but presume there must be an easier way?

**Reply by FObermaier** [reply]
Could you post the geometry definition as WKT and tell us what result you would  expext.
